src/main/webapp/css/style.css

```diff
@@ -65,6 +65,10 @@
 
 .gameButton {
     margin: none;
+    box-shadow: 0 0 1px 0
+        inset, 0 0 0 1px;
+    color: white;
+    -webkit-text-stroke: 0.3px black;
```

Comment:
I think the numbers inside the grid are too small and the stroke isn't very visible.

Reply:
<img width="1440" alt="Screen Shot 2020-07-27 at 9 13 29 PM" src="https://user-images.githubusercontent.com/49408972/88607917-73613b80-d04e-11ea-9583-487b5e4f105c.png">


I increased it a bit, let me know if its okay

Reply:
I just fixed it!